common/src/main/resources/conf/development.conf.json

```diff
@@ -1,5 +1,4 @@
 {
   "environment": "DEVELOPMENT",
-  "odbURI": "wss://odb.gpp.lucuma.xyz/ws",
-  "exploreDBURI": "wss://explore-hasura.herokuapp.com/v1/graphql"
+  "odbURI": "wss://lucuma-odb-p-unset-jlpak1dtmcd.herokuapp.com/ws"
```

Comment:
Should we have a nae under `lucuma.xyz`

Reply:
We already have it for staging: https://odb.gpp.lucuma.xyz. I was using a review app here since the required ODB changes weren't merged yet, but they are now.